### PR TITLE
chore(contract_manager): add scripts and changes to upgrade wh guardi…

### PR DESCRIPTION
**Pull request by ali-bahjati**
…an set

This change adds `sync_wormhole_guardian_set.ts` script to update all of our evm and cosmwasm contracts and has a couple of fixes in different places to make sure everything works fine.

### CODE REVIEW

File: contract_manager/package.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
   "dependencies": {
     "@certusone/wormhole-sdk": "^0.9.8",
     "@coral-xyz/anchor": "^0.29.0",
-    "@injectivelabs/networks": "1.0.68",
+    "@injectivelabs/networks": "^1.14.6",
```
Comment (ali-bahjati) [inline contract_manager/package.json:26]: we use this library to get injective grpc endpoints. It's not ideal that it's not defined explicitly and we should later make it explicit.

File: governance/xc_admin/packages/xc_admin_common/src/chains.ts [hunk 2]
```diff
@@ -53,7 +53,7 @@ export const RECEIVER_CHAINS = {
   zetachain: 60034,
   astar_zkevm: 60035,
   coredao: 60036,
-  tomochain: 60037,
+  viction: 60037,
```
Comment (ali-bahjati) [inline governance/xc_admin/packages/xc_admin_common/src/chains.ts:56]: this network is renamed.

File: contract_manager/store/contracts/EvmPriceFeedContracts.yaml [hunk 3]
```diff
@@ -169,9 +166,6 @@
 - chain: coredao_testnet
   address: "0x8D254a21b3C86D32F7179855531CE99164721933"
   type: EvmPriceFeedContract
-- chain: tomochain_testnet
```
Comment (ali-bahjati) [inline contract_manager/store/contracts/EvmPriceFeedContracts.yaml:172]: this network is renamed to victious but their testnet is a reset. so we need to redeploy later.

File: target_chains/cosmwasm/tools/src/ci/deployer/injective.ts [hunk 4]
```diff
@@ -37,7 +37,13 @@ export class InjectiveDeployer implements Deployer {
 
   private async signAndBroadcastMsg(
     msg: Msgs,
-    fee = DEFAULT_STD_FEE
+    fee: {
```
Comment (ali-bahjati) [inline target_chains/cosmwasm/tools/src/ci/deployer/injective.ts:40]: drive-by: injective package updates

File: contract_manager/src/chains.ts [hunk 5]
```diff
@@ -410,9 +410,11 @@ export class EvmChain extends Chain {
     // Some networks like Filecoin do not support the normal transaction type and need a type 2 transaction.
     // To send a type 2 transaction, remove the ``gasPrice`` field and add the `type` field with the value
     // `0x2` to the transaction configuration parameters.
+    console.log("Sending transaction with params", txParams);
     return transactionObject.send({
-      gas: gasEstimate * GAS_ESTIMATE_MULTIPLIER,
-      gasPrice: await this.getGasPrice(),
+      // gas: gasEstimate * GAS_ESTIMATE_MULTIPLIER,
```
Comment (m30m) [inline contract_manager/src/chains.ts:415]: Is this intentional?

File: contract_manager/src/contracts/cosmwasm.ts [hunk 6]
```diff
@@ -52,6 +52,7 @@ export class WormholeCosmWasmContract extends WormholeContract {
 
   async getCurrentGuardianSetIndex(): Promise<number> {
     const config = await this.getConfig();
+    console.log(config);
```
Comment (m30m) [inline contract_manager/src/contracts/cosmwasm.ts:55]: ?

**Review by m30m — APPROVED** [review]
